pilot/pkg/networking/util/util.go

```diff
@@ -71,6 +71,18 @@ const (
 	IstioMetadataKey = "istio"
 	// The range of LoadBalancingWeight is [1, 128]
 	maxLoadBalancingWeight = 128
+
+	// EnvoyTransportSocketMetadataKey is the key under which metadata is added to an endpoint
+	// which determines the endpoint level transport socket configuration.
+	EnvoyTransportSocketMetadataKey = "envoy.transport_socket"
+
+	// EnvoyRawBufferSocket matched with hardcoded built-in Envoy transport name which determines
+	// endpoint level plantext transport socket configuration
+	EnvoyRawBufferSocket = "raw_buffer"
+
+	// TLSSocketName matched with hardcoded built-in Envoy transport name which determines endpoint
+	// level tls transport socket configuration
+	TLSSocketName = "tls"
```

Comment:
be consistent of `EnvoyRawBufferSocket ` name above? `EnvoyTlsSocketName/EnvoyRawBufferSocketName`